@xen-orchestra/lite/src/App/Pools/PoolNetwork.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Map } from 'immutable'
+import { withState } from 'reaclette'
+
+import { Network, ObjectsByType, Pif } from '../../libs/xapi'
+import { FormattedMessage } from 'react-intl'
+
+interface ParentState {
+  objectsByType: ObjectsByType
+}
+
+interface State {}
+
+interface Props {
+  poolId: string
+}
+
+interface ParentEffects {}
+
+interface Effects {}
+
+interface Computed {
+  networks?: Map<string, Network>
+  objectsFetched: boolean
+  PIFs?: Map<string, Pif>
+}
+
+const Table = styled.table`
+  border: 1px solid #333;
+  td  {
+    border: 1px solid #333;
+  }
+  thead {
+    background-color: #333;
+    color: #fff;
+  }
+`
```

Comment:
IMHO, we should avoid using HTML elements directly and use dedicated components from our internal components lib, like `Table` or (`SortedTable`), `List`, etc.

We should take inspiration from BootStrap, Java, GTK, etc.